release-notes/5.2/5.2.1.md

```diff
@@ -0,0 +1,102 @@
+# Release Notes
+
+## [Stable release 5.2.1] - 2024-05-28
+
+This update brings the below changes over the previous release:
+
+### Changed
+
+- Upgraded `Azure.Identity` version from 1.10.3 to 1.11.3 [#2492](https://github.com/dotnet/SqlClient/pull/2492), [#2528](https://github.com/dotnet/SqlClient/pull/2528)
+- Upgraded `Microsoft.Identity.Client` version from 4.56.0 to 4.60.3 [#2492](https://github.com/dotnet/SqlClient/pull/2492)
+- Upgraded `Azure.Core` version from 1.35.0 to 1.38.0 [#2492](https://github.com/dotnet/SqlClient/pull/2492)
+- Code Health improvements: [#2467](https://github.com/dotnet/SqlClient/pull/2467)
+
+### Fixed
+
+- Fixed connection errors on Linux when Data Source property contains both named instance and port [#2436](https://github.com/dotnet/SqlClient/pull/2436)
+- Fixed `SqlConnection.FireInfoMessageEventOnUserErrors` when set to true throws an exception [#2505](https://github.com/dotnet/SqlClient/pull/2505)
+- Fixed exception when using `DATETIMEOFFSET(n)` in a TVP if `n` is 1, 2, 3, or 4 [#2506](https://github.com/dotnet/SqlClient/pull/2506)
+- Reverted PR [#1983](https://github.com/dotnet/SqlClient/pull/1938) which caused connection failure delays when using `OpenAsync` [#2507](https://github.com/dotnet/SqlClient/pull/2507)
+- Fixed `SqlConnection.Clone()` to include `AccessTokenCallback` [#2527](https://github.com/dotnet/SqlClient/pull/2527)
+
+## Target Platform Support
+
+- .NET Framework 4.6.2+ (Windows x86, Windows x64)
+- .NET 6.0+ (Windows x86, Windows x64, Windows ARM64, Windows ARM, Linux, macOS)
+- .NET Standard 2.0+ (Windows x86, Windows x64, Windows ARM64, Windows ARM, Linux, macOS)
+
+### Dependencies
+
+#### .NET Framework
+
+- Microsoft.Data.SqlClient.SNI 5.2.0 
+- Azure.Identity 1.11.3
+- Microsoft.Identity.Client 4.60.3
+- Microsoft.IdentityModel.Protocols.OpenIdConnect 6.35.0
+- Microsoft.IdentityModel.JsonWebTokens 6.35.0
+- System.Buffers 4.5.1
+- System.Configuration.ConfigurationManager 6.0.1
+- System.Runtime.InteropServices.RuntimeInformation 4.3.0
+- System.Text.Encoding.Web 6.0.0
+
+#### .NET 6
+
+- Microsoft.Data.SqlClient.SNI 5.2.0
```

Comment:
All target frameworks except .NET Framework should be linked with `Microsoft.Data.SqlClient.SNI.runtime`.
```suggestion
- Microsoft.Data.SqlClient.SNI.runtime 5.2.0
```